pallets/nft/src/lib.rs

```diff
@@ -409,8 +410,9 @@ pub mod pallet {
 			// TODO: if metadata can change?
 			let metadata = class_info.metadata;
 
+			let next_token_id = orml_nft::Pallet::<T>::next_token_id(class_id);
```

Comment:
Should use return value from `orml_nft::Pallet::<T>::mint` to save one call

Reply:
Thanks! This change is more applicable! Change made for all three types.